packages/std/src/results/cosmos_msg.rs

```diff
@@ -149,6 +149,15 @@ pub enum WasmMsg {
         /// msg is the json-encoded MigrateMsg struct that will be passed to the new code
         msg: Binary,
     },
+    /// Sets a new admin (for migrate) on the given contract.
+    /// Fails if this contract is not currently admin of the target contract.
+    UpdateAdmin {
+        contract_addr: String,
+        admin: Option<String>,
```

Comment:
```suggestion
        admin: String,
```

We created ClearAdmin to avoid the optionality here

Reply:
Yup, my bad.